src/cli.ts

```diff
@@ -7,6 +9,9 @@ import SentryCli, {Releases} from '@sentry/cli';
  */
 let cli: Releases;
 export const getCLI = (): Releases => {
+  // Set the User-Agent string.
+  process.env['SENTRY_PIPELINE'] = `action-release/${version}`;
```

Comment:
Hmmm, maybe `github-action-release`? Feel like that will make it obvious that it's coming from a github action.